pkg/gardenadm/cmd/connect/options.go

```diff
@@ -5,23 +5,73 @@
 package connect
 
 import (
+	"fmt"
+	"os"
+	"strings"
+
 	"github.com/spf13/pflag"
 
 	"github.com/gardener/gardener/pkg/gardenadm/cmd"
 )
 
+// Flow:
+// $ make kind-single-node-up gardenadm-up
+// $ make gardenadm-up SCENARIO=connect
+// Run `gardenadm init` on machine pod
+// $ export KUBECONFIG=dev-setup/kubeconfigs/virtual-garden/kubeconfig
+// $ go run ./cmd/gardenadm token create --validity=8h --print-connect-command --shoot-namespace=garden --shoot-name=root
+//   ^ this command needs to write the shoot namespace/name into the description of the bootstrap token secret
+//     this way, we can check in the to-be-created "shoot authorizer" if the CSR (created with the bootstrap token) has the correct gardener.cloud:system:shoot:<namespace>:<name> username
+//     this prevents claiming a client certificate for another shoot
+// $ gardenadm connect --bootstrap-token <token> --ca-certificate <ca-cert> <control-plane-address>
+
```

Comment:
I am surprised to find this in `pkg/gardenadm/cmd/connect/options.go`. Why is it here?

Reply:
Oops, that's a leftover. Thanks for spotting!